### DISCUSSION TITLE
Gravatar avatar fetch email address must be lowercased before hashing

### DISCUSSION THREAD

**Discussion by PaulMEdwards** [npm/feedback#462] [Website] [answered]
Gravatar forces email addresses input on their platform to be lowercase.

![2021-06-25](https://user-images.githubusercontent.com/153847/123480927-e30ddc80-d5b7-11eb-88b5-467b0a385584.png)

They consequently insist email address lookups are lowercased before hashing:
http://en.gravatar.com/site/implement/hash/

It appears that the NPM website is not lowercasing the email address before hashing it to submit the Gravatar lookup, otherwise mine would be working:

![2021-06-25 (1)](https://user-images.githubusercontent.com/153847/123480894-dab5a180-d5b7-11eb-93bd-8dfc3b32e029.png)

Please correct this discrepancy.
Thanks in advance!

Additional references:
- https://github.com/mhutchie/vscode-git-graph/issues/369
- https://github.com/theforeman/foreman/pull/812

**Comment by PaulMEdwards** [comment, marked as answer]
The Gravatar is now working... Did you fix the lookup or did it just take a significant amount of time for it to work?

**Reply by MylesBorins** [reply]
tbh I'm not sure 😇 perhaps there was a delay in the caching layer?
